vendor_fixes/k8s.io/apimachinery/pkg/api/apitesting/roundtrip/roundtrip.go

```diff
@@ -0,0 +1,415 @@
+/*
```

Comment:
Please add this and other files to a `vendor` directory as well. This file has diff with what I have added to `vendor`. Please use the version which is taken from k8s.io/apimachinery v0.0.0-20191004074956-01f8b7d1121a

Reply:
I am not sure if I understand this comment. Are you saying apimachinary copy in vendor is not the same as the version specified in mod?

When I run `go mod vendor` roundtrip.go doesn't change. However, there are other changes:
```
	deleted:    vendor/github.com/google/btree/btree_mem.go
	deleted:    vendor/golang.org/x/sys/unix/mkasm_darwin.go
	deleted:    vendor/golang.org/x/sys/unix/mkpost.go
	deleted:    vendor/golang.org/x/sys/unix/mksyscall.go
	deleted:    vendor/golang.org/x/sys/unix/mksyscall_aix_ppc.go
	deleted:    vendor/golang.org/x/sys/unix/mksyscall_aix_ppc64.go
	deleted:    vendor/golang.org/x/sys/unix/mksyscall_solaris.go
	deleted:    vendor/golang.org/x/sys/unix/mksysnum.go
	deleted:    vendor/golang.org/x/sys/unix/types_aix.go
	deleted:    vendor/golang.org/x/sys/unix/types_darwin.go
	deleted:    vendor/golang.org/x/sys/unix/types_dragonfly.go
	deleted:    vendor/golang.org/x/sys/unix/types_freebsd.go
	deleted:    vendor/golang.org/x/sys/unix/types_netbsd.go
	deleted:    vendor/golang.org/x/sys/unix/types_openbsd.go
	deleted:    vendor/golang.org/x/sys/unix/types_solaris.go
	deleted:    vendor/golang.org/x/text/unicode/bidi/gen.go
	deleted:    vendor/golang.org/x/text/unicode/bidi/gen_ranges.go
	deleted:    vendor/golang.org/x/text/unicode/bidi/gen_trieval.go
	deleted:    vendor/golang.org/x/text/unicode/norm/maketables.go
	deleted:    vendor/golang.org/x/text/unicode/norm/triegen.go
	deleted:    vendor/golang.org/x/text/width/gen.go
	deleted:    vendor/golang.org/x/text/width/gen_common.go
	deleted:    vendor/golang.org/x/text/width/gen_trieval.go
	deleted:    vendor/golang.org/x/tools/go/gcexportdata/main.go
	deleted:    vendor/golang.org/x/tools/imports/mkindex.go
	deleted:    vendor/golang.org/x/tools/imports/mkstdlib.go
	deleted:    vendor/k8s.io/apimachinery/pkg/api/apitesting/fuzzer/valuefuzz_test.go
	deleted:    vendor/k8s.io/apimachinery/pkg/api/validation/objectmeta_test.go
	deleted:    vendor/k8s.io/apimachinery/pkg/api/validation/path/name.go
	deleted:    vendor/k8s.io/apimachinery/pkg/api/validation/path/name_test.go
	deleted:    vendor/k8s.io/apimachinery/pkg/apis/meta/v1/validation/validation_test.go
```